src/tasks/actions/thunks.ts

```diff
@@ -411,7 +431,12 @@ export const updateScript = () => async (
       throw new Error(resp.data.message)
     }
 
-    dispatch(goToTasks())
+    if (currentPage === 'TasksPage') {
+      dispatch(goToTasks())
+    } else if (currentPage === 'TaskRunsPage') {
+      dispatch(goToTaskRuns())
+    }
```

Comment:
If we use the enum described above, this would turn into:
```ts
    if (currentPage === TaskPage.TasksPage) {
      dispatch(goToTasks())
    } else if (currentPage === TaskPage.TaskRunsPage) {
      dispatch(goToTaskRuns())
    }
```

this way, we have two cases for the two enum values and we probably don't need to define a fallback function